applications/nrf5340_audio/src/modules/audio_sync_timer_rtc_zll.c

```diff
@@ -0,0 +1,243 @@
+/*
```

Comment:
I have not looked at this one. 

I do see the value of having this possibility for ZLL as well, but we need to add CI tests to ensure we don't break this and that is a sizable undertaking.

Reply:
oh this is oversight, this file should not be in this PR!

Reply:
removed